src/ecephys_etl/modules/align_timestamps/__main__.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+
+from allensdk.brain_observatory.argschema_utilities import \
```

Comment:
Is there any way we can get away from using `argschema_utilities`? I'm not a huge fan of having our ETL packages depend on the SDK. I also don't like the idea of having a way to handle arguments/schemas outside of unmodified `argschema`.